jest.config.js

```diff
@@ -2,6 +2,7 @@
  * @type {jest.InitialOptions}
  */
 module.exports = {
+  setupFilesAfterEnv: ["<rootDir>/tests/_setup.ts"],
```

Comment:
Happy to reduce the scope of this PR by factoring out the TS test utils/setup herein to a different PR that we can merge first.